### ISSUE TITLE
ctc_beam_search_decoder

### ISSUE THREAD

**Issue by zealerww** [Belval/CRNN#22]
https://github.com/Belval/CRNN/blob/063349508bd95e8dfb6b2e1135a7f8d8dd1fbc39/CRNN/crnn.py#L167

acutually, i think you shoud set the param merge_repeated to False, so the result will keep the repeated char. `decoded, log_prob = tf.nn.ctc_beam_search_decoder(logits, seq_len, merge_repeated=False) `.

so your Ground truth text in readme.md 'semantically'  will not be recognized as 'semanticaly'

**Comment by Belval** [comment]
Closing as this was fixed in another issue https://github.com/Belval/CRNN/blob/master/CRNN/crnn.py#L167

Sorry this one fell off my radar.
